packages/client/src/components/ui/button/button.tsx

```diff
@@ -6,14 +6,16 @@ type ButtonProps = {
     buttonSize?: 'small' | 'medium' | 'large';
     buttonStyle?: 'normal' | 'outlined';
     onClick?: MouseEventHandler;
+    type?: 'button' | 'submit' | 'reset' | undefined;
 };
 const Button: FC<ButtonProps> = ({
     children,
     buttonSize = 'small',
     buttonStyle = 'normal',
     onClick,
+    type,
 }) => (
-    <button onClick={onClick} className={`${style[buttonStyle]} ${style[buttonSize]}`}>
+    <button onClick={onClick} type={type} className={`${style[buttonStyle]} ${style[buttonSize]}`}>
```

Comment:
Мне кажется вам стоит начать использовать https://www.npmjs.com/package/clsx, в конкретном примере можно и без него, но в перспективе он вам точно понадобится.
```suggestion
    <button onClick={onClick} type={type} className={clsx(style[buttonStyle], style[buttonSize])}>
```

Reply:
Вижу что ниже используется classnames. Лучше использовать его везде.